src/pages/impact.js

```diff
@@ -151,6 +166,7 @@ var Resources = React.createClass({
             </p>
             <p>
               {this.context.intl.formatMessage({id: 'scientist_librarian_article_3_fight1'})}
+              {' '}
```

Comment:
I think you can replace `{' '}` with `&nbsp;` but I have no strong opinions on which we use.

Reply:
Yeah, I prefer avoiding non breaking spaces in this case, we don’t want the last word to stick with the first word.
I kept yours in the home page because I think it’s appropriate in this context (better to have the learn more link starting on the same line), but here we don’t want to change the text flow in the middle of a paragraph

Reply:
Ah yes that makes sense.

For some reason I thought you went with `{' '}` because of some weird react quirk or something. But that makes complete sense.